src/components/ui/Text.tsx

```diff
@@ -488,3 +488,17 @@ export function HighlightedText({
 }) {
   return <div className={highlightedText(bold, center)}>{children}</div>
 }
+
+const trashButtonText = classnames(
+  textColor(
+    'text-secondary',
+    'hover:text-secondary-light-active-transparent',
```

Comment:
What about mobile hover? After the button is pressed the hover may remain

Reply:
Fixed